sphericalpolygon/excess_area.py

```diff
@@ -29,7 +29,7 @@ def polygon_excess(vertices):
         dlon = np.abs(pdlon) 
         
         # If two adjacent vertices are close enough(coincident), do nothing. 
-        if dlon < 1e-6: continue 
+        if dlon < 1e-6 and np.abs(pdlat): continue 
```

Comment:
```suggestion
        if dlon < 1e-6 and np.abs(pdlat) < 1e-6: continue 
```

Reply:
did you mean this?

Reply:
Thank you for the correction. Yes, that's what I meant.